MAESTRO/scATAC_Genescore.py

```diff
@@ -38,26 +39,26 @@ def genescore_parser(subparsers):
         "If the format is 'h5' or 'plain', users need to specify the name of the count matrix file "
         "and row names should be like 'chromosome_peakstart_peakend', such as 'chr10_100020591_100020841'. "
         "If the format is 'mtx', the 'matrix' should be the name of .mtx formatted matrix file, such as 'matrix.mtx'.")
-    group_input.add_argument("--feature", dest = "feature", default = "peaks.bed", 
+    group_input.add_argument("--feature", dest = "feature", default = "", 
         help = "Location of feature file (required for the format of 'mtx'). "
         "Features correspond to row indices of count matrix. "
-        "The feature file should be the peak bed file with 3 columns. DEFAULT: peaks.bed.")
-    group_input.add_argument("--barcode", dest = "barcode", default = "barcodes.tsv", 
+        "The feature file should be the peak bed file with 3 columns. For example, peaks.bed.")
+    group_input.add_argument("--barcode", dest = "barcode", default = "", 
```

Comment:
@taoliu `default=""` is to facilitate the initialization of the `config.yaml` file. Otherwise, we need to set the default value (for not used option) to `""` manually.